### DISCUSSION TITLE
staleTime config in TypeScript

### DISCUSSION THREAD

**Discussion by gtoshinakano** [TanStack/query#1294] [General] [answered]
Hello, I'm working on converting my application from JS to TS and, in some cases, I use react-query with `{staleTime: "Infinity"}`. In every place where staleTime is Infinity, the linter flags me because types.d.ts from node_modules says that staleTime can only be a number. 

If I change types.d.ts in node_modules to 
`
/**
     * The time in milliseconds after data is considered stale.
     * If set to `Infinity`, the data will never be stale.
     */
    staleTime?: number | 'Infinity';
`

It removes the flags.

Am I doing right?

**Comment by TkDodo** [comment, marked as answer]
`{ staleTime: Infinity }` should work. `Infinity`, without the quotes, is a number provided by JavaScript :)

**Reply by gtoshinakano** [reply]
Thanks man!! I didn't know that!
